azurecompute/src/test/java/org/jclouds/azurecompute/AzureComputeProviderMetadataLive.java

```diff
@@ -44,7 +45,7 @@ public AzureComputeProviderMetadataLive() {
 
    public static Properties defaultProperties() {
       final Properties properties = AzureManagementApiMetadata.defaultProperties();
-      properties.setProperty(TEMPLATE, "osFamily=UBUNTU,osVersionMatches=.*14\\.10,loginUser=jclouds,"
+      properties.setProperty(TEMPLATE, "osVersionMatches=1[01234].[01][04],os64Bit=true,loginUser=jclouds,"
```

Comment:
Oh, now I see you override the location here. 

Instead of hardcoding it, though, would it make sense to honor the "jclouds.region" property and put its value here, using the current one as the default? This will make easier to run the tests against a concrete location.